notebooks/CLI-examples/cli_cloud_LeNet5_training.ipynb

```diff
@@ -0,0 +1,1704 @@
+    "\n",
+    "Additional details on the RPU configuration can be found at https://aihwkit.readthedocs.io/en/latest/using_simulator.html#rpu-configurations\n",
+    "\n",
+    "For this particular case we will define a RPU which uses RRAM devices in the cross-point array:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "id": "CGNLDA2IzHeZ"
+   },
+   "outputs": [],
+   "source": [
+    "def create_rpu_config():\n",
+    "\n",
+    "    from aihwkit.simulator.presets import ReRamSBPreset\n",
+    "\n",
+    "    rpu_config=ReRamSBPreset()\n",
+    "\n",
+    "    return rpu_config"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "soEwkJg3zHeZ"
+   },
+   "source": [
+    "We can now use this rpu_config as input of the network model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "id": "oBu6VxsJzHea"
+   },
+   "outputs": [],
+   "source": [
+    "from torch.nn import Tanh, MaxPool2d, LogSoftmax, Flatten\n",
+    "from aihwkit.nn import AnalogConv2d, AnalogLinear, AnalogSequential\n",
+    "\n",
+    "def create_analog_lenet5_network():\n",
+    "    \"\"\"Returns a LeNet5 inspired analog model.\"\"\"\n",
+    "\n",
+    "    N_CLASSES = 10\n",
+    "    rpu_config = create_rpu_config()\n",
+    "    channel = [16, 32, 512, 128]\n",
+    "    model = AnalogSequential(\n",
+    "        AnalogConv2dMapped(in_channels=1, out_channels=channel[0], kernel_size=5, stride=1,\n",
+    "                     rpu_config=rpu_config),\n",
+    "        Tanh(),\n",
+    "        MaxPool2d(kernel_size=2),\n",
+    "        AnalogConv2dMapped(in_channels=channel[0], out_channels=channel[1], kernel_size=5, stride=1,\n",
+    "                     rpu_config=rpu_config),\n",
+    "        Tanh(),\n",
+    "        MaxPool2d(kernel_size=2),\n",
+    "        Tanh(),\n",
+    "        Flatten(),\n",
+    "        AnalogLinearMapped(in_features=channel[2], out_features=channel[3], rpu_config=rpu_config),\n",
+    "        Tanh(),\n",
+    "        AnalogLinearMapped(in_features=channel[3], out_features=N_CLASSES, rpu_config=rpu_config),\n",
+    "        LogSoftmax(dim=1)\n",
+    "    )\n",
+    "\n",
+    "    return model\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "EW8NXVY1zHeb"
+   },
+   "source": [
+    "## Setup the BasicTraining Experiment\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "id": "HXd7VR9szHeb"
+   },
+   "outputs": [],
+   "source": [
+    "DATASET = FashionMNIST\n",
+    "MODEL = create_analog_lenet5_network()\n",
+    "BATCH_SIZE = 8\n",
+    "LOSS_FUNCTION = NLLLoss\n",
+    "EPOCHS = 2\n",
+    "LEARNING_RATE = 0.05\n",
+    "MAX_ELEMENTS = 2000\n",
+    "\n",
+    "my_experiment = BasicTraining(\n",
```

Comment:
You call BasicTraining twice in this example. Please remove one instance.

Reply:
Done